lib/kernel/src/socket.erl

```diff
@@ -3700,7 +3715,7 @@ With argument `Cont`; equivalent to
 [`sendto(Socket, Data, Cont, infinity)`](`sendto/4`) *since OTP 24.0*.
 """.
 -spec sendto(Socket :: term(), Data :: term(), Cont :: select_info()) -> _;
-            (Socket :: term(), Data :: term(), Dest :: term()) -> _.
+            (Socket :: term(), Data :: term(), Dest :: sockaddr()) -> _.
```

Comment:
the return type is too generic (`_` which is the same as `term()`).

Can we get more detailed return type (that was in OTP 26)?

Reply:
It seems that overloaded specs are not needed here. A single spec can be used.